### PR TITLE
2 outreach code

### LINKED ISSUE #2: Outreach Code
funyon robot needs code because it is lost.

### PR DESCRIPTION
**Pull request by Glorfindel29**
Code for outreach events done code. needs merged. @Carl4 @BrendanFish can you review this?

### CODE REVIEW

**Review by Glorfindel29 — COMMENTED** [review]
Looks like the right code just waiting on approval

**Review by jmez27 — APPROVED** [review]

**Pull request merged**
